tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/GacCommandTests.cs

```diff
@@ -96,11 +99,94 @@ public void GacCompleteTest()
         // GET
         using (var consoleGet3 = ConsoleHelper.Redirect())
         {
-            var result = Program.Main($"gac get {assemblyName}".Split(' '));
+            var result = Program.Main($"gac get {DummyAssemblyName}".Split(' '));
 
             // it shouldn't be in the gac
             result.Should().NotBe(0);
-            consoleGet3.Output.Should().Contain($"Error getting '{assemblyName}' from the GAC");
+            consoleGet3.Output.Should().Contain($"Error getting '{DummyAssemblyName}' from the GAC");
+        }
+    }
+
+    [SkippableFact]
+    [Trait("RunOnWindows", "True")]
+    public void GacWithMultipleVersions()
+    {
+        Skip.IfNot(FrameworkDescription.Instance.IsWindows());
+
+        // We skip the test if we don't have permissions
+        Skip.IfNot(Gac.AdministratorHelper.IsElevated);
```

Comment:
Have you double checked this test is definitely running in CI? I assume it is, as we have other tests in here, but might be a good idea to make sure! 😄

Reply:
Yup 
<img width="1188" alt="image" src="https://github.com/user-attachments/assets/79673619-cf42-4853-ba7e-017a81993013" />

Reply:
<img width="948" alt="image" src="https://github.com/user-attachments/assets/931bf258-57f3-452e-b214-197e870de1e4" />